### DISCUSSION TITLE
Low Resolution

### DISCUSSION THREAD

**Discussion by GFA-SkyRanger** [kibook/pmms#47] [Q&A]
Hi,
is there any way to increase the resolution?
all videos, regardless if yt oder a 4k file, is very pixelate.
at least 720 HD would be enought. but it seems to be around 640x480 or something.

**Comment by kibook** [comment]
If you're using render targets (playing stuff on like a TV screen in-game), the maximum resolution is limited by the render target texture, and most of the ones in the base game are very low resolution.

I think I heard of someone replacing the texture with their own, higher resolution one, but it's not something I have tried myself. In theory, that could work though.

You can also get a higher resolution image by using a 3D scaleform screen instead of a render target.

There's a second limit to the resolution which is the size of the DUI browser displaying the media, and that is set to 1280x720 by default. You can increase it in [config.lua](https://github.com/kibook/pmms/blob/fc6b0fdcf38c2c84fe05fc27ada3089236ae96a3/config.lua#L725-L729), the only thing to keep in mind is you'll have to re-adjust any scaleform screen settings you have saved to account for the new base resolution.
